Resources/Prototypes/_NF/Recipes/Lathes/weapons.yml

```
@@ -94,73 +85,65 @@
 # Melee weapons
 - type: latheRecipe
   id: BaseBallBatNF
+  parent: BaseWeaponRecipeLong
   result: BaseBallBat
-  category: Weapons
-  completetime: 4
   materials:
     Wood: 700
 
 - type: latheRecipe
   id: BolaNF
+  parent: BaseWeaponRecipe
   result: Bola
-  category: Weapons
-  completetime: 2
   materials:
     Durathread: 200
     Plasteel: 600
 
 - type: latheRecipe
   id: KukriKnifeNF
+  parent: BaseWeaponRecipe
   result: KukriKnife
-  category: Weapons
-  completetime: 4
   materials:
     Plasteel: 500
     Plastic: 200
 
 - type: latheRecipe
   id: MacheteNF
+  parent: BaseWeaponRecipe
   result: Machete
-  category: Weapons
-  completetime: 4
   materials:
     Steel: 500
     Plastic: 200
 
 - type: latheRecipe
   id: KatanaNF
+  parent: BaseWeaponRecipeLong
   result: Katana
-  category: Weapons
-  completetime: 6
   materials:
     Plasteel: 500
     Plastic: 300
 
 - type: latheRecipe
   id: WeaponCrusher
+  parent: BaseWeaponRecipe
```

Did you want a change from 5 to 2 seconds?
```suggestion
  parent: BaseWeaponRecipeLong
```